src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/CommandContext.java

```diff
@@ -18,35 +18,56 @@
 import org.openkilda.messaging.Message;
 
 import lombok.Data;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
```

Comment:
It's OK as a quick solution. But this bounds CommandContext to classes of the transport layer (kafka). All arguments constructor + a dedicated builder may handle the case better.